websites/C/Canvas/presence.ts

```diff
@@ -704,9 +704,8 @@ presence.on("UpdateData", async () => {
 												timeElapsed.getTime() + daySeconds * 1000
 											);
 											delete presenceData.startTimestamp;
-											presenceData.endTimestamp = Math.floor(
-												timeElapsed.getTime() / 1000
-											);
+											[presenceData.startTimestamp, presenceData.endTimestamp] =
+												presence.getTimestamps(elapsed, duration);
```

Comment:
Due to the way this presence works, this should probably just set the start timestamp to `Date.now() / 1000` in addition to the previous `endTimestamp` code